lesson16/main.js

```diff
@@ -0,0 +1,119 @@
+"use strict";
+
+const jsonUrl = "https://jsondata.okiba.me/v1/json/yFXrE210903000829";
+const newsWrapper = document.getElementById("js-news");
+
+const createElementWithClassName = (element, name) => {
+  const variableName = document.createElement(element);
```

Comment:
IMO
I think it is createdElement that is created from createElement
```suggestion
  const createdElement = document.createElement(element);
```

Reply:
Right, `variable = element` is weird. I fix it.